fix devhelp 3.8.2 prefs dialog bug

kuletco opened the pull request:
1. fix devhelp prefs dialog bug(notebook not expand)
2. generate window icon to $prefix/share/devhelp/devhelp.ico
3. hide control window when start devhelp.exe directly in the windows.

Alexpux commented on .gitattributes:
```diff
@@ -8,12 +8,15 @@
 *.msi binary
 *.rar binary
 *.so  binary
+*.lib binary
```
Duplicate. See some lines below

Alexpux commented on mingw-w64-asciidoctor-extensions/PKGBUILD:
```diff
@@ -0,0 +1,52 @@
+# Maintainer: nalla <nalla@hamal.uberspace.de>
+
+_realname=asciidoctor-extensions
+
```
Add pkgbase entry

Alexpux commented on mingw-w64-devhelp/PKGBUILD:
```diff
@@ -12,6 +12,7 @@ depends=("${MINGW_PACKAGE_PREFIX}-gtk3"
          "${MINGW_PACKAGE_PREFIX}-gsettings-desktop-schemas"
```
Increase pkgrel.

mati865 commented:
179 commits?!

kuletco commented:
I am so sorry bring you these troubles.
This is my first time using github to submit codes, I only modified one project named devhelp and submit it.
I didn't know why its happen.

This is my steps:
1. Clone codes from your master branch(https://github.com/Alexpux/MINGW-packages.git)
2. I modified somethings about devhelp.
3. I fork the branch to my account
4. I modified the remote repository url to my forked url on my local repository(git remote set-url origin "my url").
5. I add relationship to your branch on my local repository(git remote add upstream https://github.com/Alexpux/MINGW-packages.git)
6. I check modified(git status) and commit to my forked repository. At this step, only one file modified and one file to add.
7. I started a "new pull request".

Then I don't know why the question appear.

Please check and help me to solve the problems, thanks.

mati865 commented:
I think you have cloned or referenced this repository https://github.com/git-for-windows/MINGW-packages

kuletco commented:
I think you are right, but now how can I pull my modified to you?
Delete my fork and fork, clone, modified, commit, and start pull again?

mati865 commented:
Hmm, you could create new branch and hard reset it to https://github.com/Alexpux/MINGW-packages.git.
Then just cherry pick your commits from master branch.

mati865 commented:
Forgot to say you will have to create new PR.

kuletco commented:
Thank you, I will doing again based on your suggestion.

kuletco commented:
I have create a new PR
